v1.1-alpha.20170601.md

```diff
@@ -0,0 +1,72 @@
+- The Kubernetes templates now use the latest production release instead of tracking testing releases. [#15878](https://github.com/cockroachdb/cockroach/pull/15878)
+
+### SQL Language Changes
+
+- The `information_schema.user_privileges` table is now supported. [#15745](https://github.com/cockroachdb/cockroach/pull/15745)
+- The `ON UPDATE RESTRICT` and `ON DELETE RESTRICT` modifiers can now be used when creating foreign keys (this behavior is the default, but now it can be specified explicitly). [#15815](https://github.com/cockroachdb/cockroach/pull/15815)
+- The hash functions `sha512()`, `fnv32()`, `fnv32a()`, `fnv64()`, `fnv64a()`, `crc32ieee()`, and `crc32c()` are now supported. All hash functions now accept multiple arguments and arguments of type `BYTES`. [#15828](https://github.com/cockroachdb/cockroach/pull/15828) [#15859](https://github.com/cockroachdb/cockroach/pull/15859)
+- Casts to type `DECIMAL` can now specify precision and scale. [#15834](https://github.com/cockroachdb/cockroach/pull/15834)
+- A new aggregate function `xor_agg()` is now available. [#15831](https://github.com/cockroachdb/cockroach/pull/15831)
+- Schema change commands now wait until the schema change is complete instead of starting it in the background. [#15598](https://github.com/cockroachdb/cockroach/pull/15598)
+- It is now possible to `SELECT` from the result of a `SHOW` statement with the syntax `SELECT ... FROM [ SHOW ... ] WHERE ...` [#15590](https://github.com/cockroachdb/cockroach/pull/15590)
+- It is now possible to `CREATE` a table and `ALTER` (or `DROP`) it in the same transaction. [#15929](https://github.com/cockroachdb/cockroach/pull/15929)
+
+### Command-Line Interface Changes
+
+- A new `--advertise-port` flag is supported to go along with `--advertise-port`. [#15700](https://github.com/cockroachdb/cockroach/pull/15700)
+- A `raw` mode has been added to the SQL shell's `display_format` options. [#15590](https://github.com/cockroachdb/cockroach/pull/15590)
+
+### Admin UI Changes
+
+- The log viewer now shows multi-line log entries correctly. [#15949](https://github.com/cockroachdb/cockroach/pull/15949)
+- A new query plan inspector is available at `/queryplan`. [#15908](https://github.com/cockroachdb/cockroach/pull/15908)
+
+### Bug Fixes
+
+- `MAX(bool)` and `MIN(bool)` no longer panic. [#15803](https://github.com/cockroachdb/cockroach/pull/15803)
+- `SHOW GRANTS *` on an empty database now works correctly. [#15881](https://github.com/cockroachdb/cockroach/pull/15881)
+- Fixed a data race involving transaction records. [#15882](https://github.com/cockroachdb/cockroach/pull/15882)
+- `DECIMAL` types now report the correct precision and scale for compatibility with JDBC. [#15927](https://github.com/cockroachdb/cockroach/pull/15927)
+
+### Performance Improvements
+
+- Incremental `BACKUP` is now much faster. [#15004](https://github.com/cockroachdb/cockroach/pull/15004)
```

Comment:
This is off by default (see comments on #15004). Need to remove this line when updating these notes.